releng/platform.target

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?><target name="ELS Target" sequenceNumber="14">

```

Comment:
ls-api is correctly pushed to a p2 repo: http://services.typefox.io/open-source/jenkins///job/lsapi/lastSuccessfulBuild/artifact/build/p2-repository/ , see https://github.com/eclipselabs/eclipse-language-service/blob/master/pom.xml#L39 . The remaining issue is gson 2.5. But since ls-api is moving to Eclipse.org, GSon 2.5 will have to go to Orbit, and we'll have a good portable TP soon.

However, at the moment, I prefer calling it `ide.target` to avoid confusion. Later, we can rename it to platform.target and use it in both IDE and Tycho.